### DISCUSSION TITLE
German Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by web-dev-sam** [johnlindquist/kit#1478] [Q&A]
I have a german keyboard and the shown shortcuts make nooo sense. It wants me to press `Ctrl+;` but to write ";" I have to press `Shift+,` but then `Ctrl+,` and `Ctrl+Shift+,` dont work. So now I had to press every imaginable shortcut to find the right one and in the process I made the app smaller. Now I dont know how to make it bigger again because `Ctrl++` doesnt work. The shown shortcuts should adapt to the keyboard otherwise its almost impossible to use properly. So basically it looks like its inaccesssible to anybody not using an english keyboard.

Is there atleast a setting I can change to do something about it?

**Comment by web-dev-sam** [comment]
Yea thats an uninstall from me sadly. I may return after a proper UX/UI overhaul.

**Comment by JosXa** [comment]
I'm also on a German keyboard. If you're on Windows, just click the Kit icon in the taskbar and choose `Change shortcut`:
![image](https://github.com/johnlindquist/kit/assets/7313176/900cd865-3c14-46c3-a01b-cf55144a4510)

In fact I got it configured to use `Ctrl+ä` for the main shortcut and that works perfectly. It'll add `KIT_MAIN_SHORTCUT=control ä` to your `~/.kenv/.env`.

May wanna give it another shot? :)

**Reply by JosXa** [reply]
@johnlindquist I do agree however that `Ctrl+;` is absolutely impossible to type on many non-QWERTY keyboard layouts. I think it should prompt you once at startup which keys you'd like to use.
